### ISSUE TITLE
Redo on tag level 

### ISSUE THREAD

**Issue by Discostu36** [MarcusWolschon/osmeditor4android#1147]
Description:
---

When accidentally tapping "undo" in the tag editing window, there is no "redo" to get the data back. This could be a long touch option for the button.

Device info:
---
<table>
<tr><td>App version</td><td>14.0.19.0</td></tr>
<tr><td>App version code</td><td>1321</td></tr>
<tr><td>Android build version</td><td>A750FNXXS4BTB1</td></tr>
<tr><td>Android release version</td><td>9</td></tr>
<tr><td>Android SDK version</td><td>28</td></tr>
<tr><td>Android build ID</td><td>PPR1.180610.011.A750FNXXS4BTB1</td></tr>
<tr><td>Device brand</td><td>samsung</td></tr>
<tr><td>Device manufacturer</td><td>samsung</td></tr>
<tr><td>Device name</td><td>a7y18lte</td></tr>
<tr><td>Device model</td><td>SM-A750FN</td></tr>
<tr><td>Device product name</td><td>a7y18ltexx</td></tr>
<tr><td>Device hardware name</td><td>samsungexynos7885</td></tr>
<tr><td>ABIs</td><td>[arm64-v8a, armeabi-v7a, armeabi]</td></tr>
<tr><td>ABIs (32bit)</td><td>[armeabi-v7a, armeabi]</td></tr>
<tr><td>ABIs (64bit)</td><td>[arm64-v8a]</td></tr>
</table>

**Comment by simonpoole** [comment]
This is something we will be looking at in https://github.com/MarcusWolschon/osmeditor4android/issues/808 Currently there is no real "undo" in the property editor it simply reinstates what was there before starting it. We may move this to the more general undo mechanism.

**Comment by simonpoole** [comment]
Superseded by https://github.com/MarcusWolschon/osmeditor4android/issues/1493
